src/app/platform/components/sidebar.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { Library, Plus } from "lucide-react";
+import LogoMark from "./navbar.svg";
+import Image from "next/image";
+const sidebarItems = [
+  { name: "List", icon: Library },
+];
+
+const Sidebar = () => {
+  const [activeItem, setActiveItem] = useState("List");
+
+  return (
+    <aside className="h-screen bg-gray-900 text-white lg:w-64 w-16 transition-all duration-300">
+      <div className="flex items-center justify-center py-4 lg:py-6">
+        <Image
```

Comment:
Don't think the logo is required here because we are already rendering the logo on the Navbar